docs/chrome-flags-for-tools.md

```diff
@@ -94,6 +94,10 @@ These flags are being used in various tools. They also just need to be documente
 --disable-features=site-per-process # Disables OOPIF. https://www.chromium.org/Home/chromium-security/site-isolation
 --disable-hang-monitor
 
+--disable-backgrounding-occluded-windows
```

Comment:
seems like it shouldn't be in both lists?

Reply:
the md is just for other tool developers and hackers like vikram who are digging into this. ;)

Reply:
oh I see now :)